cmd/verify/verify.go

```diff
@@ -0,0 +1,216 @@
+package verify
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrKubectlInstall  = errors.New("kubectl is not installed")
+	ErrClusterInfo     = errors.New("set the kubeconfig or kubeconfig does not have required access")
+	ErrArgoCD          = errors.New("argocd is not installed")
+	ErrArgoCDAuthToken = errors.New("argocd auth token has expired, login to argocd again")
+	ErrGit             = errors.New("git is not installed")
+	ErrArlonNs         = errors.New("arlon is not installed")
+	ErrCapi            = errors.New("capi services are not installed")
+	ErrCapiCP          = errors.New("error fetching the capi cloudproviders")
+)
+
+func NewCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:               "verify",
+		Short:             "Verify if arlonctl can be run",
+		Long:              "Verify if required kubectl,argocd,git access is present before profiles and bundles are created",
+		DisableAutoGenTag: true,
+		Example:           "arlonctl verify",
+		RunE: func(c *cobra.Command, args []string) error {
+			// Verify kubectl status
+			kubectlStatus, err := verifyKubectl()
+			if err != nil {
+				fmt.Println("Error while verifying kubectl status: ", err)
+			} else {
+				fmt.Println("Successfully verified kubectl status")
+			}
+
+			// Verify argocd status
+			argoStatus, err1 := verifyArgoCD()
```

Comment:
Why declare new variables `err1` ... `errN`? You can just reuse `err`.

Reply:
Will do